image-layout.md

```diff
@@ -1,13 +1,13 @@
 ## Open Container Initiative Image Layout Specification
 
-The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
```

Comment:
On Mon, Aug 29, 2016 at 03:02:46PM -0700, Brandon Philips wrote:
> I don't know. I simply don't understand why having a directory
> called refs/tags is confusing.

You'll have people talking about refs and people talking about tags
and people wondering if there's some distinction between them (which
it sounds like there isn't).

> What is your use case for having lots of refs pointing to different
> images in a single image-layout?

I'm publishing a bunch of images on example.com, and don't want to be
bothered with maintaining multiple image-layout instances [1].  Or I
want a convenient way to distribute my public key along with the
images it signs [2].  And supporting generic refs protects
image-layout implementations from having to worry about scope creep
and such if a more opinionated refs spec decided that it needed to
support something different from the narrow use case it initially
picked.

> rkt image import --name mystuff.com/foo/bar /path/to/oci-image-layout
> docker import --name mystuff.com/foo/bar /path/to/oci-image-layout
> 
> In these cases the name and tags make sense. What is your use case?

I'd rather keep both explicit:

  $ my-tool import LAYOUT-PATH REF [NAME]

unless we decide to support something like HEAD [1], in which case REF
would also be optional.  If name was not specified on the command
line, the name used for the imported object would come from some
verifiable name (e.g. using the Merkle objects discussed in #176, see
[3]).

[1]: https://github.com/opencontainers/image-spec/pull/219#issuecomment-243197974
[2]: https://github.com/opencontainers/image-spec/pull/219#discussion_r76691546
[3]: https://github.com/opencontainers/image-spec/issues/173#issuecomment-234640781